prisma/schema.prisma

```diff
@@ -61,6 +61,8 @@ model User {
     SeniorIDs     String[]  @db.ObjectId
     Seniors       Senior[]  @relation(fields: [SeniorIDs], references: [id])
     approved      Approval  @default(PENDING)
+    ChapterID     String    @db.ObjectId
+    Chapter       Chapter   @relation(fields: [ChapterID], references: [id])
```

Comment:
Great addition! There's a `1 - many` mapping between `Chapter` and `User`. One question we should ask is that can a `User` exist without a `Chapter`? What does the current schema tell you?